cfg/supervision-128k.cfg

```diff
@@ -33,5 +33,5 @@ SEGMENTS {
     ZEROPAGE: load = RAM,      type = bss, define   = yes;
     DATA:     load = RAM,      type = bss, define   = yes, offset = $0200;
     BSS:      load = RAM,      type = bss, define   = yes;
-    VECTOR:   load = ROM,      type = ro,                  offset = $3FFA;
+    VECTORS:  load = ROM,      type = ro,                  start  = $3FFA;
```

Comment:
According to https://cc65.github.io/doc/ld65.html#ss5.7 the `offset` is relative to the start of the memory area. The `ROM` memory area starts at $C000. So the effective address of this section was $FFFA (which nicely matches the 6502 hardware ;-). In contrast the `start` is an absolute value - so at least from my POV this needs to read
```
start = $FFFA
```